### PR TITLE
Moved the initial CMPC repository

### PR DESCRIPTION
**Pull request by P2P-Nathan**

### CODE REVIEW

File: Index.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+[
```
Comment (P2P-Nathan) [inline Index.json:1]: The index has been re-named as Index.json and is currently (and should remain) in alphabetical order.

Comment (clarkd) [reply Index.json:1]: Interested to understand why it must be alphabetical? Seems like that could get out of sync quickly and could surely be resolved by ordering when you retrieve the file etc.?

**Review by richbenwell — APPROVED** [review]

File: Index.json (hunk 1 above)
Comment (richbenwell) [reply Index.json:1]: Helps maintainability

Comment (P2P-Nathan) [reply Index.json:1]: RB had suggested it on a previous revision, has no purpose to the code but it helps readability for quick additions, or to find missing items as the folders are sorted alphabetically as well.

File: Aquilaweb.Support.PendingReboot/details.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "ManagementPackSystemName": "Aquilaweb.Support.PendingReboot",
+  "ManagementPackDisplayName": "Aquilaweb Support - Pending Reboots,
```
Comment (DavidR91) [inline Aquilaweb.Support.PendingReboot/details.json:3]: There is a missing quote at the end of this line

(I know this is merged already but this issue still exists in master: https://github.com/squaredup/scom-community-catalog/blob/master/Aquilaweb.Support.PendingReboot/details.json and the open PR as well)

**Pull request merged**